theme-contract/src/main/java/com/orange/ouds/theme/tokens/semantic/OudsGridTokens.kt

```diff
@@ -0,0 +1,63 @@
+/*
+ * Software Name: OUDS Android
+ * SPDX-FileCopyrightText: Copyright (c) Orange SA
+ * SPDX-License-Identifier: MIT
+ *
+ * This software is distributed under the MIT license,
+ * the text of which is available at https://opensource.org/license/MIT/
+ * or see the "LICENSE" file for more details.
+ *
+ * Software description: Android library of reusable graphical components
+ */
+
+package com.orange.ouds.theme.tokens.semantic
+
+import androidx.compose.runtime.Stable
+import androidx.compose.ui.unit.Dp
+import androidx.compose.ui.unit.dp
+import com.orange.ouds.theme.AdaptiveTokenValue
+import com.orange.ouds.theme.OudsAdaptiveDisplayType
+import com.orange.ouds.tokens.global.raw.GridRawTokens
+
+class OudsGridTokens(
+    var minWidth: AdaptiveTokenValue = AdaptiveTokenValue(GridRawTokens.gridMinWidthExtraCompact.dp, GridRawTokens.gridMinWidthCompact.dp, GridRawTokens.gridMinWidthMedium.dp),
```

Comment:
We should use `val` like what we did in `OudsColorTokens` otherwise someone might change theme tokens. We also need to do this for border, elevation and opacity semantic tokens.